open-metadata-resources/open-metadata-archives/open-metadata-types/src/main/java/org/odpi/openmetadata/opentypes/OpenMetadataTypesArchive.java

```diff
@@ -217,6 +218,1635 @@ private ClassificationDef addTemplateClassification()
+        this.archiveBuilder.addTypeDefPatch(updateCloudPlatformClassification());
+        this.archiveBuilder.addTypeDefPatch(updateCloudTenantClassification());
+        this.archiveBuilder.addTypeDefPatch(updateCloudServiceClassification());
+        this.archiveBuilder.addTypeDefPatch(updateCommentEntity());
+        this.archiveBuilder.addTypeDefPatch(updateGraphStoreEntity());
+        this.archiveBuilder.addTypeDefPatch(updateLogFileEntity());
+        this.archiveBuilder.addTypeDefPatch(updateDatabaseEntity());
+        this.archiveBuilder.addTypeDefPatch(updateDatabaseServerClassification());
+        this.archiveBuilder.addTypeDefPatch(updateMetadataRepositoryEntity());
+        this.archiveBuilder.addTypeDefPatch(updateMetadataServerClassification());
+        this.archiveBuilder.addTypeDefPatch(updateRepositoryProxyClassification());
+        this.archiveBuilder.addTypeDefPatch(updateBusinessCapabilityEntity());
+        this.archiveBuilder.addTypeDefPatch(updateDataStoreEntity());
+        this.archiveBuilder.addTypeDefPatch(updateDataSourcePhysicalStatusAnnotationEntity());
+        this.archiveBuilder.addTypeDefPatch(updateEnterpriseAccessLayerEntity());
+        this.archiveBuilder.addTypeDefPatch(updateMetadataCollectionEntity());
+        this.archiveBuilder.addTypeDefPatch(updateExternalReferenceEntity());
+        this.archiveBuilder.addTypeDefPatch(updatePropertyFacetEntity());
+        this.archiveBuilder.addTypeDefPatch(updateCohortMemberEntity());
+        this.archiveBuilder.addTypeDefPatch(updateImplementationSnippetEntity());
+        this.archiveBuilder.addTypeDefPatch(updatePolicyAdministrationPointClassification());
+        this.archiveBuilder.addTypeDefPatch(updatePolicyDecisionPointClassification());
+        this.archiveBuilder.addTypeDefPatch(updatePolicyEnforcementPointClassification());
+        this.archiveBuilder.addTypeDefPatch(updatePolicyInformationPointClassification());
+        this.archiveBuilder.addTypeDefPatch(updatePolicyRetrievalPointClassification());
+    }
+
+
+    /**
+     * Deprecate clashing properties and add new ones to replace them.
+     * @return the type def patch
+     * @see #updateSoftwareServerPlatformEntity()
+     */
+    private TypeDefPatch updateSoftwareServerPlatformEntity()
+    {
+        /*
+         * Create the Patch
+         */
+        final String typeName = "SoftwareServerPlatform";
+
+        TypeDefPatch  typeDefPatch = archiveBuilder.getPatchForType(typeName);
+
+        typeDefPatch.setUpdatedBy(originatorName);
+        typeDefPatch.setUpdateTime(creationDate);
+
+        /*
+         * Build the attributes
+         */
+        List<TypeDefAttribute> properties = new ArrayList<>();
+        TypeDefAttribute       property;
+
+        final String attribute1Name            = "type";
+        final String attribute1Description     = "Deprecated attribute. Use the platformType attribute to describe the type of software server platform.";
```

Comment:
Should this description refer to deployedImplementationType rather than platformType?

Reply:
thanks for catching all of these! I've updated in local branch and will open a new PR that includes Mandy's other merged changes.